packages/features/ee/teams/components/MemberListItem.tsx

```diff
@@ -78,11 +117,11 @@ export default function MemberListItem(props: Props) {
     },
   });
 
-  const ownersInTeam = () => {
-    const { members } = props.team;
-    const owners = members.filter((member) => member["role"] === MembershipRole.OWNER && member["accepted"]);
-    return owners.length;
-  };
+  // const ownersInTeam = () => {
+  //   const { members } = props.team;
+  //   const owners = members.filter((member) => member["role"] === MembershipRole.OWNER && member["accepted"]);
+  //   return owners.length;
+  // };
```

Comment:
```suggestion

```

Reply:
@joeauyeung Actually i am not able to understand what is the use of total number of owners https://github.com/calcom/cal.com/pull/16155/files#r1714008461